package.json

```diff
@@ -85,6 +85,6 @@
   "license": "MIT",
   "readmeFilename": "README.md",
   "engines": {
-    "node": ">=6"
+    "node": ">=8"
```

Comment:
Can you remove this change? It's not correct since it's still possible to run on Node 6 if you install the correct version of Knex as listed in `peerDependencies`.

Reply:
yep. I put it back.